### PR TITLE
EVG-2661 separate database from rest of config in evergreen service

### PR DESCRIPTION
**Pull request by john-m-liu**
This change is non-functional until I resolve the TODO comment, but to do that I first need to make sure the evergreen config we have in chef explicitly has the /etc/mci_settings file set

BUILD-4576

It looks like we already have that set, so I've removed the default

### CODE REVIEW

File: operations/flags.go [hunk 1]
```diff
@@ -144,6 +153,41 @@ func addMigrationRuntimeFlags(flags ...cli.Flag) []cli.Flag {
...
+		},
+		cli.BoolFlag{
+			Name:  dbFsyncFlagName,
+			Usage: "True if pending writes should flush to disk",
+		},
+		cli.BoolFlag{
+			Name:  dbJournalAckFlagName,
+			Usage: "True if the journal must acknowledge writes",
+		},
+	)
```
Comment (tychoish) [inline operations/flags.go]: Let's not give people the choice to turn off fsync/journal (fsync doesn't make sense after mmap) 

also the URL should be setable via an environment variable defined here.

Comment (tychoish) [reply operations/flags.go]: also we should make sure that these options, if set in the mongodb uri have clear semantics when they're specified by flags.

Comment (john-m-liu) [reply operations/flags.go]: I removed fsync/journal and also let the MONGO_URL env variable override the default URL

It looks like mgo doesn't support setting the write concern with the URL so I left the current write concern flags as is

Comment (tychoish) [reply operations/flags.go]: https://docs.mongodb.com/manual/reference/connection-string/#write-concern-options

Comment (john-m-liu) [reply operations/flags.go]: Let's keep it like this for now for backwards compatibility, and I'll open a ticket to switch this to use just the connection string

**Pull request merged**